Tentacle/Issue.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  Issue.swift
+//  Tentacle
+//
+//  Created by Romain Pouclet on 2016-05-23.
+//  Copyright © 2016 Matt Diephouse. All rights reserved.
+//
+
+import Argo
+import Curry
+
+/// An Issue on Github
+public struct Issue: Hashable, CustomStringConvertible {
+    public enum State: String {
+		case Open = "open"
+        case Closed = "closed"
+	}
+	
+    public let id: Int
```

Comment:
This should be `ID` to match the existing types and should use a `String` instead of an `Int`, even though it's numeric. I made that decision because the API on the whole is a bit inconsistent.